cdqa/utils/converters.py

```diff
@@ -123,7 +123,7 @@ def generate_squad_examples(question, closest_docs_indices, metadata, retrieve_b
     return squad_examples
 
 
-def pdf_converter(directory_path):
+def pdf_converter(directory_path, line_paragraph=False):
```

Comment:
Honestly, I don't know if this is a good name for this arg... It's not obvious to understand it by just its name. At the same time, it's pretty difficult to find a proper name for it. Maybe something like `include_line_breaks`? 

I am open to suggestions.

If you can also add a docstring to explain the function and the args it would be very useful too :)

Reply:
I honestly tried and couldn't think of anything better. So `include_line_breaks` it is!
:)

Reply:
perfect!